src/components/Dashboard/Utilization/UtilizationItem.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { Col, Row, SparklineChart } from 'patternfly-react';
+
+import { InlineLoading } from '../../Loading';
+
+export const UtilizationItem = ({ id, title, data, maxY, unit, isLoading, LoadingComponent }) => {
+  const actual = !isLoading && Math.round(data[data.length - 1]);
+
+  const axis = {
+    y: {
+      show: true,
+      padding: {
+        top: 0,
+        bottom: 0,
+      },
+      tick: {
+        count: 3,
+      },
+    },
+    x: {
+      show: false,
+    },
+  };
+  if (maxY) {
+    axis.y.max = maxY;
+  }
+
+  const chartData = {
+    columns: [[unit, ...data.map(val => val.toFixed(1))]],
+    unload: true,
+    type: 'area',
+  };
+
+  return (
+    <div className="kubevirt-utilization__item">
+      <Row>
+        <Col lg={2} md={2} sm={2} xs={2}>
+          {title}
+        </Col>
+        <Col className="kubevirt-utilization__item-actual" lg={3} md={3} sm={3} xs={3}>
+          {isLoading || `${actual} ${unit}`}
+        </Col>
+        <Col className="kubevirt-utilization__item-chart" lg={7} md={7} sm={7} xs={7}>
+          {isLoading ? <LoadingComponent /> : <SparklineChart id={id} data={chartData} axis={axis} unloadBeforeLoad />}
```

Comment:
can you add 'Not available' state ? when isLoading is false and data are null. We already have that in `ClusterCapacity` https://github.com/kubevirt/web-ui-components/blob/master/src/components/Dashboard/Capacity/CapacityItem.js#L24

Reply:
done